roles/manage-jira/tasks/create_project.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create Sample Project
+  uri:
+    url: "{{ atlassian.jira.url }}/rest/api/2/project"
+    method: POST
+    user: "{{ atlassian.jira.username }}"
+    password: "{{ atlassian.jira.password }}"
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+    header:
+      - Accept: 'application/json'
+      - Content-Type: 'application/json'
+    body: "{ 'key': '{{ atlassian.jira.key }}',
+             'name': '{{ atlassian.jira.project }}',
```

Comment:
I'd suggest keeping the variable names in the dictionary the same as the payload - i.e.: in this case `project` --> `name`, or if name is too generic, maybe use `project_name` or more preferably another layer in the dict - i.e.: `atlassian.jira.project.name`